storm-core/src/native/worker-launcher/impl/main.c

```diff
@@ -269,6 +274,45 @@ int main(int argc, char **argv) {
       return INVALID_ARGUMENT_NUMBER;
     }
     exit_code = signal_container_as_user(user_detail->pw_name, container_pid, signal);
+  } else if (strcasecmp("run-oci-container", command) == 0) {
```

Comment:
nit: why not the standard strcmp()?

Reply:
I don't have strong opinion on making it case-sensitive. But we have been using case-insensitive in other commands here, don't think of changing it. case-insensitive is fine with me.